File: Algorithm_class/Codes/Python/n_ci_sade.py
n = int(input())
say, eded = 0, 1
while say < n:
    eded += 1
    yoxla = True
    for i in range(2, int(eded**0.5) + 1):
        if eded % i == 0:
            yoxla = False
            break
    if yoxla: say += 1
print(eded)

File: Algorithm_class/Codes/C++/n_ci_sade.cpp
#include <iostream>
using namespace std;
int main() {
    int n; cin >> n;
    int say = 0, eded = 1;
    while (say < n) {
        eded++;
        bool sadedir = true;
        for (int i = 2; i * i <= eded; i++) {
            if (eded % i == 0) { sadedir = false; break; }
        }
        if (sadedir) say++;
    }
    cout << eded << endl;
    return 0;
}
